coriolis/tests/test_policy.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2023 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from unittest import mock
+
+from coriolis import policy
+from coriolis.tests import test_base
+
+
+class PolicyTestCase(test_base.CoriolisBaseTestCase):
+    """Test suite for the Coriolis policy module."""
+
+    @mock.patch.object(policy, '_ENFORCER')
+    def test_reset(self, mock_enforcer):
+        result = policy.reset()
+
+        mock_enforcer.clear.assert_called_once_with()
+        self.assertEqual(result, None)
```

Comment:
Please also add a `self.assertEqual(policy._ENFORCER, None)`

Reply:
Great! I added , thank you!